src/pages/Listings.js

```diff
@@ -0,0 +1,32 @@
+import React, { Component } from "react";
+// import { Redirect } from 'react-router-dom';
+import NavtabTop from "../components/NavtabTop";
+import NavtabBottom from "../components/NavtabBottom";
+import Footer from "../components/Footer";
+import Listing from "../components/Listing";
+import Sort from "../components/Sort";
+import "../App.css";
+
+class Home extends Component {
```

Comment:
Same comment as before.. this seems like a typo. Does this component belong in this PR...since it was part of the previous PR (#10 )